tests/virt/node/migration_and_maintenance/test_post_copy_migration.py

```
@@ -111,20 +112,22 @@ def drained_node_with_hotplugged_vm(admin_client, hotplugged_vm):
             id="WIN-VM",
-            marks=[pytest.mark.special_infra, pytest.mark.high_resource_vm],
+            marks=[pytest.mark.special_infra, pytest.mark.high_resource_vm, pytest.mark.x86_64],
         ),
     ],
     indirect=True,
 )
 class TestPostCopyMigration:
     @pytest.mark.dependency(name=f"{TESTS_CLASS_NAME}::migrate_vm")
     @pytest.mark.polarion("CNV-11421")
+    @pytest.mark.s390x
     def test_migrate_vm(self, hotplugged_vm, vm_background_process_id, migrated_hotplugged_vm):
```

Ruff (0.14.4)

Unused method argument: `migrated_hotplugged_vm`

(ARG002)